Extensions/ArtifactEngine/Engine/cilogger.ts

```diff
@@ -0,0 +1,21 @@
+var tl = require('vsts-task-lib');
+
+const area: string = 'artifact-engine';
+
+function getDefaultProps() {
+    return {
+        releaseuri: tl.getVariable('Release.ReleaseUri'),
+        releaseid: tl.getVariable('Release.ReleaseId'),
+        builduri: tl.getVariable('Build.BuildUri'),
+        buildid: tl.getVariable('Build.Buildid')
+    };
+}
+
+export function publishEvent(feature, properties: { [key: string]: any }): void {
+    try {
+        tl.assertAgent('2.125.0');
+        tl.publishTelemetry(area, feature, Object.assign(getDefaultProps(), properties));
```

Comment:
if download fails on lower agent versions, it will hinder us from going forward. we need telemetry to know what issues are getting it. do you suggest we should run the config change again? or should we log telemetry using the older way?